examples/multi_run_simulator.py

```diff
@@ -58,15 +57,15 @@ def run_many_inline(parameter_set_list, n_threads=None, progress_bar=True):
     else:
         progress_monitor = lambda x: x
 
-    # Create a pool and evaluate models concurrently
-    with ThreadPool(processes=n_threads) as pool:
-
+    with ProcessPoolExecutor(n_threads) as ex:
```

Comment:
NB: I think we could also use `multiprocessing.pool.Pool` which is process based rather than thread based and seems to be basically equivalent to what I've used here. However this `concurrent.futures` library is simpler and designed for Python3, it is also more explicit about using processes, so I think it's better to use this.

Reply:
I prefer `concurrent.futures`